src/main/java/org/generationcp/middleware/dao/oms/CVTermDao.java

```diff
@@ -130,9 +131,109 @@ public CVTerm getByCvIdAndDefinition(final Integer cvId, final String definition
 		return termIds;
 	}
 
+	public Map<String, MeasurementVariable> getVariablesByNamesAndVariableType(final List<String> names, final VariableType variableType) {
+		final Map<String, MeasurementVariable> stdVarMap = new HashMap<>();
+
+		// Store the names in the map in uppercase
+		for (int i = 0, size = names.size(); i < size; i++) {
+			names.set(i, names.get(i).toUpperCase());
+		}
```

Comment:
We should not modify the passed argument, this can be simplified to:
final List<String> namesUppercase = names.stream().map(String::toUpperCase).collect(Collectors.toList());